File: backend/spring/src/main/java/io/ssafy/openticon/controller/request/EditDeviceTokenRequestDto.java
package io.ssafy.openticon.controller.request;

import jakarta.persistence.Column;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@AllArgsConstructor
@NoArgsConstructor
public class EditDeviceTokenRequestDto {
    @Column(nullable = false)
    private String deviceToken;
    @Column(nullable = false)
    private boolean mobile;
}

File: android/app/src/main/java/io/ssafy/openticon/ui/data/model/EditDeviceTokenRequestDto.kt
package io.ssafy.openticon.ui.data.model

data class EditDeviceTokenRequestDto(
    val deviceToken: String, // 필수 필드로 nullable이 아니게 설정
    val mobile: Boolean      // 필수 필드로 nullable이 아니게 설정
)
